src/GraphQL/DI/IServiceRegister.cs

```diff
@@ -25,7 +27,9 @@ public interface IServiceRegister
         /// in case of <see cref="RegistrationCompareMode.ServiceTypeAndImplementationType"/>)
         /// has not already been registered.
         /// </summary>
-        IServiceRegister TryRegister(Type serviceType, Type implementationType, ServiceLifetime serviceLifetime, RegistrationCompareMode mode = RegistrationCompareMode.ServiceType);
+        IServiceRegister TryRegister(Type serviceType,
+            [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)]
+            Type implementationType, ServiceLifetime serviceLifetime, RegistrationCompareMode mode = RegistrationCompareMode.ServiceType);
```

Comment:
```suggestion
        IServiceRegister TryRegister(
            Type serviceType,
            [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] Type implementationType,
            ServiceLifetime serviceLifetime,
            RegistrationCompareMode mode = RegistrationCompareMode.ServiceType);
```

Reply:
Code style suggestion - for multi-argument signatures with new attributes place each argument on its own line.

Reply:
i think that works